README.md

```diff
@@ -4,13 +4,174 @@
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 
-## Introduction
+# Introduction
 
-Sciuro is a bridge between alertmanager and Kubernetes to sync alerts as
-Node Conditions. It is designed to work in tandem with other controllers
-that observe Node Conditions such as [draino](https://github.com/planetlabs/draino) or the [cluster-api](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html).
+Sciuro is a bridge between alertmanager and Kubernetes to sync alerts as Node
+Conditions. It is designed to work in tandem with other controllers that
+observe Node Conditions such as [draino](https://github.com/planetlabs/draino)
+or the [cluster-api](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html).
 
-## Requirements
+# Requirements
 
 * Alertmanager API v2
 * Kubernetes 1.12+
+
+# Deployment
+1. Download the manifests from the latest Github release
+
+```
+wget https://github.com/cloudflare/sciuro/releases/latest/download/cluster.yaml
+wget https://github.com/cloudflare/sciuro/releases/latest/download/stable.yaml
+```
+
+2. Apply the cluster-scoped resources that allow Sciuro to read nodes and
+   modify their status. If you choose a different namespace, adjust the
+   namespace name and `ClusterRoleBinding` accordingly.
+
+```
+# Review manifests and make adjustments for different namespace
+kubectl apply -f cluster.yaml
```

Comment:
You can kubectl these addresses directly.

```
kubectl apply -f https://github.com/cloudflare/sciuro/releases/latest/download/cluster.yaml
```

Reply:
True, but I think its better to encourage people to read the manifests. In addition the configmap in the `stable.yaml` has to be modified so might as well keep things symmetrical.